assets/queries/openAPI/link_object_operation_id_does_not_target_an_operation_object/query.rego

```diff
@@ -23,6 +23,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	op := doc.components.responses[r].links[l].operationId
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)